### DISCUSSION TITLE
Perfect hit causes pp reduction (catch)

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#28557] [Q&A] [answered]
https://github.com/ppy/osu/assets/34775378/54135c6d-a5ca-49a2-8f28-3b0f5e284d82

can show that pp reduced after a perfect catch

This is more obvious in difficult diff, but it also happens in normal diff (reduce about 1pp).

map in video:
https://osu.ppy.sh/beatmapsets/660630

I don't know if this is some kind of problem, because perfect hit should never cause pp reduce

**Comment by BlockHead007** [comment, marked as answer]
It is normal for calculating sliders, it's just how it works. Same thing is in stable and in osu!standard too. These pp reductions don't affect on what player gets at the end.
